problems/[230713] 733. Flood Fill.js

```diff
@@ -1,35 +1,39 @@
-/**
- * @param {number[][]} image
- * @param {number} sr
- * @param {number} sc
- * @param {number} color
- * @return {number[][]}
- */
-// 7:42 ~ 8:58
-const floodFill = function (image, sr, sc, color) {
-  const targetColor = image[sr][sc];
-  const checker = Array.from({ length: image.length }, () =>
-    Array.from({ length: image[0].length }, () => 0)
-  );
+  /**
+   * @param {number[][]} image
+   * @param {number} sr
+   * @param {number} sc
+   * @param {number} color
+   * @return {number[][]}
+   */
+  // 7:42 ~ 8:58
+  const floodFill = function (image, sr, sc, color) {
+    const targetColor = image[sr][sc];
+    const checker = Array.from({ length: image.length }, () =>
+      Array.from({ length: image[0].length }, () => 0)
+    );
 
-  function DFS(x, y) {
-    if (x >= image.length || y >= image[0].length || x < 0 || y < 0) {
-      return;
-    }
-    if (image[x][y] !== targetColor || checker[x][y] === 1) {
-      return;
-    } else {
-      checker[x][y] = 1;
-      image[x][y] = color;
+    function DFS(x, y) {
+      if (
+        x >= image.length ||
+        y >= image[0].length ||
+        x < 0 ||
+        y < 0 ||
+        image[x][y] !== targetColor ||
+        checker[x][y] === 1
+      ) {
```

Comment:
x, y 모든 위치를 다 탐색했는지 확인하기 위해서 checker를 만드신 것 같아요! 
checker 없이 image[x][y]가 color로 바뀌었는지 확인함으로써 방문여부를 판단할 수 있을 것 같아요. 
```suggestion
    function DFS(x, y) {
      if (
        x >= image.length ||
        y >= image[0].length ||
        x < 0 ||
        y < 0 ||
        image[x][y] !== targetColor ||
        image[x][y] === color
      ) {
```